src/keras_geometric/layers/pooling/attention_pooling.py

```
@@ -0,0 +1,459 @@
+                "dropout": self.dropout_rate,
+            }
+        )
+        return config
+
+    @classmethod
+    def from_config(cls, config: dict[str, Any]) -> "Set2Set":
+        """
+        Creates a Set2Set layer instance from a configuration dictionary.
+        
+        Args:
+        	config: A dictionary containing the layer configuration.
```

Ruff (0.11.9)

Indentation contains mixed spaces and tabs

(E101)